app/src/main/java/org/eyeseetea/malariacare/sdk/SdkPullController.java

```diff
@@ -261,7 +261,9 @@ private static void pullOrganisationUnits() {
                     public void call(List<OrganisationUnit> organisationUnits) {
                         Log.d(TAG, "OrganisationUnit: Done");
                         asyncDownloads--;
```

Comment:
@idelcano I think this variable management is not necessary if we invoke convertData when finish last request to api to pull events and if ConvertData is not invoked from any place more